ghc-8.4.2 fails to build due to haddock-api-2.19.0.1 requiring ghc-8.4.1

```
stack --stack-yaml=stack.yaml install                                      \
		&& cp '/Users/jky/.local/bin/hie' '/Users/jky/.local/bin/hie-8.4.2'    \
		&& cp '/Users/jky/.local/bin/hie-8.4.2' '/Users/jky/.local/bin/hie-8.4'

Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for haddock-api-2.19.0.1:
    ghc-8.4.2 from stack configuration does not match ==8.4.1  (latest matching version is 8.4.1)
needed due to haskell-ide-engine-0.2.0.0 -> haddock-api-2.19.0.1

Some different approaches to resolving this:

  * Set 'allow-newer: true' to ignore all version constraints and build anyway.

  * Consider trying 'stack solver', which uses the cabal-install solver to attempt to find some working build
    configuration. This can be convenient when dealing with many complicated constraint errors, but results may be
    unpredictable.

  * Recommended action: try adding the following to your extra-deps
    in /Users/jky/wrk/arbor-networks/haskell-ide-engine/stack.yaml:

- ghc-8.4.1

Plan construction failed.
make: *** [hie-8.4.2] Error 1
```

--- a/stack.yaml
+++ b/stack.yaml
@@ -26,8 +26,8 @@ extra-deps:
 - apply-refact-0.5.0.0
 - brittany-0.11.0.0
 - constrained-dynamic-0.1.0.0
-- haddock-api-2.19.0.1
-- haddock-library-1.5.0.1
+- haddock-api-2.20.0
+- haddock-library-1.6.0
 - haskell-lsp-0.2.2.0
 - haskell-lsp-types-0.2.2.0
 - syz-0.2.0.0

